docs/data/rpc/api-reference/methods/getLedgerEntries.mdx

```diff
@@ -48,291 +147,179 @@ print(
 )
 ```
 
-#### JavaScript
-
-If you are using the [JavaScript](https://stellar.github.io/js-stellar-sdk/) `stellar-sdk` to generate these keys, you will need to install the latest pre-release version of the SDK. This can be done like so:
-
-```bash
-yarn add @stellar/stellar-sdk
-```
-
 ```js
```

Comment:
not sure if it matters but, should this be a typescript code block like the other examples above?